CONTRIBUTING.md

```diff
@@ -2,6 +2,46 @@
 
 When contributing to this repository, please first discuss the change you wish to make via issue, or any other method with the owners of this repository before making a change.
 
+## Environment setup
+
+Being able to run tests requires an account or service principal with permission (typically Owner) in your Azure subscription.
```

Comment:
Is it necessary to have owner on an entire subscription? In my test environment I have created two Resource Groups that are dedicated for just Ansible development and testing, and I gave my configured SP the Owner role only at the scope of those two RGs. However I have not run all of the different tests, only the ones targeting modules I was directly opening PRs for, so I don't know if the more targeted scopes really are sufficient or not. For a contributing guide I would think it should be the minimum necessary permissions.

Reply:
You're correct: the permission does not need to be on the whole subscription. I will update and clarify it would only need to be Owner on the resource groups.